assets/src/edit-story/components/panels/document/pageAdvancement/pageAdvancement.js

```diff
@@ -119,47 +122,36 @@ function PageAdvancementPanel() {
       collapsedByDefault={false}
     >
       <Row>
-        <Note>
+        <MutedText>
           {__(
             'Control whether a story auto-advances between pages, or whether the reader has to manually tap to advance.',
             'web-stories'
           )}
-        </Note>
+        </MutedText>
       </Row>
       <SwitchRow>
         <Switch
-          label={__('Page Advancement', 'web-stories')}
+          groupLabel={__('Page Advancement', 'web-stories')}
+          name="page-advancement-switch"
           value={autoAdvance}
           onLabel={__('Auto', 'web-stories')}
           offLabel={__('Manual', 'web-stories')}
           onChange={updateAutoAdvance}
         />
       </SwitchRow>
       {autoAdvance && (
-        <>
-          <Row>
-            <Label>{__('Default Page Duration', 'web-stories')}</Label>
-            <RangeInput
-              min={MIN_MAX.PAGE_DURATION.MIN}
-              max={MIN_MAX.PAGE_DURATION.MAX}
-              majorStep={1}
-              minorStep={0.1}
-              value={duration}
-              handleChange={handleRangeChange}
-              aria-label={__('Default Page Duration', 'web-stories')}
-            />
-          </Row>
-          <Row>
-            <BoxedNumeric
-              symbol={_x('s', 'Seconds', 'web-stories')}
-              value={duration}
-              onChange={setDuration}
-              aria-label={__('Default page duration in seconds', 'web-stories')}
-              min={MIN_MAX.PAGE_DURATION.MIN}
-              max={MIN_MAX.PAGE_DURATION.MAX}
-            />
-          </Row>
-        </>
+        <Row>
+          <NumericInput
+            unit={` ${__('seconds', 'web-stories')}`}
+            suffix={__('Duration', 'web-stories')}
+            value={duration}
+            onChange={handleChange}
+            aria-label={__('Default page duration in seconds', 'web-stories')}
+            min={MIN_MAX.PAGE_DURATION.MIN}
+            max={MIN_MAX.PAGE_DURATION.MAX}
+            isFloat
```

Comment:
Do we want to allow floats? I saw that before we only allowed them with the slider. Since that was removed I was unsure as to whether we want integer values here or not.

Reply:
I think with a component like `NumericInput` it makes sense to allow, and make a distinction between, floats and integers.